services/web-server/src/graphql/WorkerManager.graphql

```diff
@@ -0,0 +1,25 @@
+type WMWorkerTypeSummary {
```

Comment:
`type WorkerManagerWorkerTypeSummary`. I prefer being verbose for better readability.

Reply:
wfm - I'll change this everywhere